src/main/resources/schemas/V60__smartdocuments_informatie_object_type.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+ALTER TABLE ${schema}.smartdocuments_document_creation_template
+    ADD informatie_object_type_url VARCHAR NOT NULL;
+
+COMMENT ON COLUMN ${schema}.smartdocuments_document_creation_template.informatie_object_type_url is 'URL of the Informatie Object Type';
+
+ALTER TABLE ${schema}.smartdocuments_document_creation_template
+    ADD CONSTRAINT un_sd_template_zaakafhandelparameters_id_informatie_object_type_url UNIQUE (zaakafhandelparameters_id, informatie_object_type_url);
+
+CREATE INDEX idx_document_creation_template_informatie_object_type_url
+    on ${schema}.smartdocuments_document_creation_template USING btree (informatie_object_type_url);
```

Comment:
What is a `btree`? Do we really need this? Do we already use this?

Reply:
> The name of the index method to be used. Choices are btree, hash, gist, spgist, gin, brin, or user-installed access methods like bloom. The default method is btree.

So not strictly necessary but we use it in other migrations

Reply:
If we already use it fine! I'm just a bit wary of using PostgreSQL specific stuff if it is not strictly needed.